lib/exercises.rb

```diff
@@ -1,19 +1,54 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n^2)
+# Space Complexity: o(n) => created 1 new hash and 1 new array. It should be o(2n), but we can ignor the constant
 
 def grouped_anagrams(strings)
```

Comment:
The time complexity here is O(n) because checking to see if something is in a **hash** is O(1).  That's why we use hashes.  

Otherwise 👍